ocamlformat-help.txt

```diff
@@ -44,6 +44,9 @@ OPTIONS (CODE FORMATTING STYLE)
            fit-or-vertical vertically breaks expressions if they do not fit
            on a single line. The default value is wrap.
 
+       --break-sequences
+           Break sequences.
```

Comment:
This is too terse. Perhaps something like "Force sequence expressions to break irrespective of margin."